Assets/_Scripts/Managers/CanvasManager.cs

```diff
@@ -117,9 +124,19 @@ protected override void Update()
 
         if (GameManager.Instance.Lives == 0)
         {
-            GameOverMenu.SetActive(true);
+            EndGameMenu.SetActive(true);
+            WinText.enabled = false;
+            GameOverText.enabled = true;
         }
```

Comment:
Galaga actually treats the number of `Lives` as the number of ships that the player has left to use, including the one the player is actually using.

![image](https://github.com/SolomanDavis/VGP145-S24-capstone-retro/assets/16367411/28c58cc2-8244-43c8-8c81-982e1de2f348)

You can see from this screenshot^ that it's possible to have `Lives == 0` but still be in play. It's only when you die while you have 0 lives left that the game is actually over, not when your lives drops from 1 -> 0.

This is accounted for in the `GameManager` when setting `Lives`: https://github.com/SolomanDavis/VGP145-S24-capstone-retro/blob/4d2104b110ff82e969362e843c8e2249964c9fa8/Assets/_Scripts/Managers/GameManager.cs#L23-L38

when setting, if the new value for `Lives` is less than 0, then we call `GameManager::GameOver()` [here](https://github.com/SolomanDavis/VGP145-S24-capstone-retro/blob/4d2104b110ff82e969362e843c8e2249964c9fa8/Assets/_Scripts/Managers/GameManager.cs#L55-L60) which doesn't do anything yet. I think this method should be implemented to call a `CanvasManager::GameOver()` similar to `CanvasManager::GameWon()` that you already have.

It's also nice because it means the `CanvasManager` doesn't need to check the number of lives on every update.

Reply:
Changes made.